src/core/api/thread_ftd_api.cpp

```diff
@@ -242,6 +249,14 @@ ThreadError otThreadSetSteeringData(otInstance *aInstance, otExtAddress *aExtAdd
 {
     return aInstance->mThreadNetif.GetMle().SetSteeringData(aExtAddress);
 }
+#else
+ThreadError otThreadSetSteeringData(otInstance *aInstance, otExtAddress *aExtAddress)
+{
+    (void)aInstance;
+    (void)aExtAddress;
+
+    return kThreadError_NotCapable;
```

Comment:
Should use `kThreadError_NotImplemented` here.  The `kThreadError_NotCapable` is intended for use by the Device Mode flags, where a device's code may implement a feature but the existing Device Mode does not allow it.